action/template.go

```diff
@@ -15,28 +16,34 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var err error
```

Comment:
@smothiki before, I was referring to creating the error values globally, not setting all encountered errors to a global value. See below for additional comments